bitnami/jaeger/templates/query/service.yml

```diff
@@ -0,0 +1,59 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "common.names.fullname" . }}
```

Comment:
Here you want to have a helper for the query, something like `jaeger.query.fullname`, which uses `common.names.fullname` and adds a `-query` suffix at the end. This should be applicable to the rest of the objects

Reply:
Done [1a1f238](https://github.com/bitnami/charts/pull/13480/commits/1a1f238f45d5c6699193bf2aaafdbf683e437036)